tracks/go/exercises/hamming/mentoring.md

```diff
@@ -44,8 +44,7 @@ The most common feedback revolves around:
 * Is the error message empty or vague? In Go the returned error can provide context for where the error occurred and why, so it should concisely and specifically describe the problem.
 * Is the error formatted correctly? For example the error [should not be capitalized or have punctuation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).
 * Did they use a combination of `errors.New` and `fmt.Sprintf`? Hint at `golint` and `fmt.Errorf`.
-* Did they use `fmt.Errorf` for a non-interpolated string? It may be worth suggesting they use `errors.New` in these cases to explicitly indicate that no interpolation will be occurring. Encouraging early exposure to the `errors` package might start benefitting them immediately, e.g., `errors.Wrap`.
-* Did they return `-1` or `0` in the error case? Returning the zero value may be more idiomatic, since the error indicates that the preceding value isn't that useful. There are test cases that specify `-1` should be returned, but because this value isn't checked by the tests, a student doing strict TDD might not discover this; careful readers might. 
```

Comment:
I would still keep a mention of the `-1` issue; some people will instinctively use an out-of-band value like this in the error case, because they're used to languages where this is necessary. Even though the tests no longer specify `-1`, I think we'll still see solutions where `-1` is returned, so it would be nice for the mentor notes to pick up on this.

Reply:
That's a good point, I'll reword it.